content/docs/components/serving/kfserving.md

```diff
@@ -36,24 +34,24 @@ KFServing can be installed with Kubeflow v0.7, and KFServing kustomize installat
 * [Pipelines](https://github.com/kubeflow/kfserving/tree/master/docs/samples/pipelines)
 * [Explainability](https://github.com/kubeflow/kfserving/tree/master/docs/samples/explanation/alibi)
 
-## Sample Notebooks
-* [SDK Client](https://github.com/kubeflow/kfserving/blob/master/docs/samples/client/kfserving_sdk_sample.ipynb)
-* [Transformer (Pre/Post Processing)](https://github.com/kubeflow/kfserving/blob/master/docs/samples/transformer/image_transformer/kfserving_sdk_transformer.ipynb)
+## Sample notebooks
+* [SDK client](https://github.com/kubeflow/kfserving/blob/master/docs/samples/client/kfserving_sdk_sample.ipynb)
+* [Transformer (pre/post processing)](https://github.com/kubeflow/kfserving/blob/master/docs/samples/transformer/image_transformer/kfserving_sdk_transformer.ipynb)
 * [ONNX](https://github.com/kubeflow/kfserving/blob/master/docs/samples/onnx/mosaic-onnx.ipynb)
 
-Please be on the lookout, we are constantly adding [more examples](https://github.com/kubeflow/kfserving/tree/master/docs/samples/) about available features
+We frequently add examples to our [GitHub repo](https://github.com/kubeflow/kfserving/tree/master/docs/samples/).
 
-## Learn More
-* Join our [Working Group](https://groups.google.com/forum/#!forum/kfserving) for meeting invites and discussion.
+## Learn more
+* Join our [working group](https://groups.google.com/forum/#!forum/kfserving) for meeting invitations and discussion.
 * [Read the docs](https://github.com/kubeflow/kfserving/tree/master/docs).
 * [API docs](https://github.com/kubeflow/kfserving/tree/master/docs/apis/README.md).
 * [Roadmap](https://github.com/kubeflow/kfserving/tree/master/ROADMAP.md).
 * [KFServing 101 slides](https://drive.google.com/file/d/16oqz6dhY5BR0u74pi9mDThU97Np__AFb/view).
 
 ## Prerequisites
-KNative Serving (v0.8.0 +) and Istio (v1.1.7+) should be available on Kubernetes Cluster.
+KNative Serving (v0.8.0 +) and Istio (v1.1.7+) are available on Kubernetes Cluster.
```

Comment:
It may be clearer to keep the words "should be" here. While you're here, though, please would you change "cluster" to lower-case "c" and add "your" - "on your Kubernetes cluster".

Reply:
Done :)

Reply:
Thanks!